scripts/anvil_tools/file_exists_checker.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -e
+
+# if num arguments != 2 || file does not exist || filename suffix != ".xlsx" || col number != integer
+if [ "$#" -ne 2 ] || [ ! -f "$1" ] || grep -q xlsx "$1" || [[ ! $2 =~ ^[[:digit:]]+$ ]];
+  then
+    echo "Missing required arguments or incorrect argument format/type."
+    echo "Usage: $0 [EXCEL_FILE] [COLUMN_NUMBER_TO_CHECK]"
+    echo "EXCEL_FILE: path to input excel (.xlsx) file."
+    echo "COLUMN_NUMBER_TO_CHECK: integer/number of column with gs://fc-XXXX file paths validate."
```

Comment:
1-indexed, i assume?

Reply:
not really, with bash the script name is also a "parameter, aka $0

Reply:
no i mean... if the column to check is the second column, i should have this input param be 2, not 1, right?

Reply:
yes! the column number in the excel is 1 index.